subworkflows/ebi-metagenomics/combined_gene_caller/main.nf

```diff
@@ -17,8 +16,8 @@ workflow COMBINED_GENE_CALLER {
     PRODIGAL ( ch_assembly, channel.value("sco") )
     ch_versions = ch_versions.mix(PRODIGAL.out.versions)
 
-    FRAGGENESCAN ( ch_assembly )
-    ch_versions = ch_versions.mix(FRAGGENESCAN.out.versions)
+    FRAGGENESCANRS ( ch_assembly, "illumina_5" )
```

Comment:
is this `illumina_5` always going to be consistent? could it change if we want to use the CGC on non-illumina data?

Reply:
That should be the default... I'll push that to a parameter (I forgot)